src/components/WebexMeetingControl/WebexMeetingControl.jsx

```diff
@@ -57,14 +58,19 @@ function renderButton(sc, action, display, style, showText, asItem, autoFocus, t
       </Button>
     );
   } else {
+    if (display.ID === 'mute-audio' && display.state === 'disabled' && text === 'No microphone') {
+      ariaLabelText = 'No Microphone, disabled';
```

Comment:
The text ('No Microphone, disabled', 'No Camera, disabled') is suggested by the Applause team and mentioned inside the JIRA.

Reply:
Can we put the texts in variables and then use them here. The coderabbit suggestion is also similar.
We can have two variables NO_MIC_ARIA_LABEL. NO_CAMERA_ARIA_LABEL.

Reply:
@Shreyas281299 , Code Updated!

Reply:
Would it not automatically announce `disabled` - since the state is disabled right? Is there a need to add it here also?

Reply:
@adhmenon , Yes, we need to add it here because isDisabled property is not working even if we set the value to true